roles/setup/vars/RedHat-8.yml

```diff
@@ -1,5 +1,5 @@
-additional_system_packages:
-  - python39-pip
+system_pip_package: python39-pip
```

Comment:
we should use this pattern for Debian and RHEL9 too

I think we can just set system_pip_package in defaults

```diff
diff --git a/roles/setup/defaults/main.yml b/roles/setup/defaults/main.yml
index 66b4345..8605da2 100644
--- a/roles/setup/defaults/main.yml
+++ b/roles/setup/defaults/main.yml
@@ -55,11 +55,11 @@ arch_mapping:
   x86_64: amd64
   i386: i386
 
-additional_system_packages:
-  - python3-pip
+additional_system_packages: []
 
 additional_python_packages: []
 
+system_pip_package: python3-pip
 pip_executable: pip3
 python_executable: python3
```

Reply:
good point can u just send a "suggestion" so i can click merge #lazy

Reply:
done!